rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,388 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+**Table of Contents** _generated with [DocToc](https://github.com/thlorenz/doctoc)_
+
+- [RFC: Support for components in the Fluent UI umbrella that are not core](#rfc-support-for-components-in-the-fluent-ui-umbrella-that-are-not-core)
+  - [Summary](#summary)
+  - [Problem statement](#problem-statement)
+  - [Detailed Design or Proposal](#detailed-design-or-proposal)
+    - [Deviating from v9 core concepts](#deviating-from-v9-core-concepts)
+      - [Pros](#pros)
+      - [Cons](#cons)
+    - [Breaking change cadence](#breaking-change-cadence)
+      - [Same cadence as core components](#same-cadence-as-core-components)
+        - [Pros](#pros-1)
+        - [Cons](#cons-1)
+      - [Faster than core components](#faster-than-core-components)
+        - [Pros](#pros-2)
+        - [Cons](#cons-2)
+    - [Choosing a repo for](#choosing-a-repo-for)
+      - [Current monorepo](#current-monorepo)
+        - [Pros](#pros-3)
+        - [Cons](#cons-3)
+      - [Separate monorepo](#separate-monorepo)
+        - [Pros](#pros-4)
+        - [Cons](#cons-4)
+    - [Package scope](#package-scope)
+      - [@fluentui scope](#fluentui-scope)
+        - [Pros](#pros-5)
+        - [Cons](#cons-5)
+      - [New package scope](#new-package-scope)
+        - [Pros](#pros-6)
+        - [Cons](#cons-6)
+    - [How users should consume components](#how-users-should-consume-components)
+      - [Suite packages](#suite-packages)
+        - [Pros](#pros-7)
+        - [Cons](#cons-7)
+      - [Individual packages](#individual-packages)
+        - [Pros](#pros-8)
+        - [Cons](#cons-8)
+    - [How to depend on core components](#how-to-depend-on-core-components)
+      - [Depend directly on components packages](#depend-directly-on-components-packages)
+        - [Pros](#pros-9)
+        - [Cons](#cons-9)
+      - [Depend on suite package is peer dependency](#depend-on-suite-package-is-peer-dependency)
+      - [Pros](#pros-10)
+      - [Cons](#cons-10)
+    - [Documentation](#documentation)
+      - [Current docsite](#current-docsite)
+        - [Pros](#pros-11)
+        - [Cons](#cons-11)
+      - [New docsite](#new-docsite)
+        - [Pros](#pros-12)
+        - [Cons](#cons-12)
+    - [Parnter ownership and collaboration](#parnter-ownership-and-collaboration)
+      - [GitHub](#github)
+        - [Pros](#pros-13)
+        - [Cons](#cons-13)
+      - [ADO mirroring](#ado-mirroring)
+        - [Pros](#pros-14)
+        - [Cons](#cons-14)
+      - [Both](#both)
+        - [Pros](#pros-15)
+        - [Cons](#cons-15)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# RFC: Support for components in the Fluent UI umbrella that are not core
+
+[@ling1726](https://github.com/ling1726) [@layershifter](https://github.com/layershifter)
+
+## Summary
+
+@fluentui/react-components has matured, with many flexible primitives that are also highly customizable. These
```

Comment:
Is this intended to apply over time as a principle to ALL of Fluent UI as the title implies or simply `@fluentui/react-components`? The way this is written implies both but if this is broadly applicable to Fluent UI as a whole we should probably remove anything react specific and look beyond just that scope as the requirements and implementation details will differ and pros/cons may not be broadly applicable. If the intent is solely to resolve for React, can we update the title of this RFC to do so?

Reply:
This RFC is located in the react-components folder of RFCs, for me that's pretty clear that this applies only to React packages and does not tread in the realm of any other platforms like WC

Reply:
Thanks @ling1726, @JustSlone and @levithomason as FYI to just round out our offline discussion.

My primary reasoning for asking is that so far, inevitably, some of the decisions made in isolation around React have impacted other implementations, such as the Web Components package. Thanks for the clarification. I'd be happy to add feedback here if we think this will apply more broadly in the future. If we don't, we can litigate/discuss if and when we need to.